support/k8s/k8s-workload-registrar/config_crd_test.go

```diff
@@ -0,0 +1,349 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/spiffe/spire/test/spiretest"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	testMinimalCRDConfig = `
+		trust_domain = "TRUSTDOMAIN"
+		cluster = "CLUSTER"
+		server_socket_path = "SOCKETPATH"
+		mode = "crd"
+	`
```

Comment:
nit: line break here so its easier to see where the multiline indicators start/end.